automated_packaging/prepare_new_changelog.pl

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env perl
```

Comment:
I… don't entirely understand why we need to go through this automation here. Couldn't our devs just add `CHANGELOG` entries _during_ their coding? Like, have an _unreleased_ section at the top that everyone adds to?